magicctapipe/io/gadf.py

```diff
@@ -228,6 +234,151 @@ def create_event_hdu(
     return event_hdu
 
 
+
```

Comment:
oh now I see there is another one above... VERY CONFUSING

Reply:
We had to comment the 'new' functions to update the code only up to DL1-stereo (we had to revert back some of the io/gadf functions so that now the older RFs, DL2 and DL3 can be used. I commented them because in the next PR we will delete the older version and re-integrate the newer, but I can delete them to avoid confusion (I just have to create a backup....)